.babelrc

```diff
@@ -2,9 +2,12 @@
   "plugins": [
     "transform-object-rest-spread",
     "transform-flow-comments",
+    "transform-runtime",
     "add-module-exports"
   ],
   "presets": [
-    "es2015"
+    "es2015",
```

Comment:
I had only 2017 and imports were not supported so I was assuming that they stack to enable all the features.